app/controllers/v3/service_instances_controller.rb

```diff
@@ -254,7 +254,10 @@ def create_user_provided(message)
 
   def create_managed(message, space:)
     service_plan = ServicePlan.first(guid: message.service_plan_guid)
-    unprocessable_service_plan! unless service_plan_valid?(service_plan, space)
+    unprocessable_service_plan! unless service_plan_valid?(service_plan)
+    unavailable_service_plan!(service_plan.name, service_plan.guid) unless service_plan_active?(service_plan)
+    service_plan_not_visible_in_space!(service_plan.name, service_plan.guid, space.name, space.guid) \
```

Comment:
This syntax is hard to read. I'd prefer passing `(service_plan, space)` to the method to make the line shorter and move the unless into the same line. Same for other method calls.